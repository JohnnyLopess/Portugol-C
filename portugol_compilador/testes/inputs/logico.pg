programa
{
    funcao inicio() {
        logico verdadeiro
        verdadeiro = 1
        escreva(verdadeiro)
    }
}
