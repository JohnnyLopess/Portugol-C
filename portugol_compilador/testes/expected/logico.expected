#include <stdio.h>

int main() {
    bool verdadeiro;
    verdadeiro = 1;
    printf("%d\n", verdadeiro);
    return 0;
}
